### PR TITLE
34651: GraphQL Customer business Field

### PR DESCRIPTION
**Pull request by hadidefra**

### CODE REVIEW

File: app/transformers/rural-payments-portal/customer.js [hunk 1]
```diff
@@ -73,3 +75,24 @@ export function transformNotificationsToMessages (notifications = [], showOnlyDe
...
+        extensions: {
+          code: 'FORBIDDEN'
+        }
+      }
+    )
+  }
+
+  return {
+    id: filteredBusinessForCustomer[0].id,
+    customerId: filteredBusinessForCustomer[0].id,
```
Comment (csaltmountain) [inline app/transformers/rural-payments-portal/customer.js:94]:
```suggestion
    customerId: filteredBusinessForCustomer[0].customerId,
```

`id` and `customerId` aren't the same

Comment (hadidefra) [reply app/transformers/rural-payments-portal/customer.js:94]: there is no customer id in summary in original
```
return summary.map(({ id, name, sbi }) => ({
    customerId,
    id,
    name,
    sbi
  }))

```
and in parsed you have this: 

```
async businesses ({ id }, __, { dataSources }) {
    const summary = await dataSources.ruralPaymentsPortalApi.getPersonSummaryByPersonId(id)
    return transformPersonSummaryToCustomerAuthorisedBusinesses(id, summary)
  }
```

Comment (hadidefra) [reply app/transformers/rural-payments-portal/customer.js]: done in commit: 11bd36b

**Review by jack-mccabe-defra — APPROVED** [review]
LGTM

**Pull request merged**